docs/design/coreclr/botr/ilc-architecture.md

```diff
@@ -43,12 +43,11 @@ ILC is designed to support multiple code generation backends that target the sam
 ILC currently supports following codegen backends (with varying levels of completeness):
 
 * **RyuJIT**: native code generator also used as the JIT compiler in CoreCLR. This backend supports x64, arm64, arm32 on Windows, Linux, macOS, and BSD.
-* **CppCodegen**: a portable code generator that translates CIL into C++ code. This supports rapid platform bringup (instead of having to build a code generator for a new CPU architecture, it relies on the C++ compiler the platform likely already has). Portability comes at certain costs.
```

Comment:
I would put this in a separate paragraph because it was an interesting piece of tech that will maybe be interesting again if business priorities change:

In the past, ILC supported following backends:
* **CppCodegen**: ... This codegen backend wasn't brought over from the now archived CoreRT(link) repo.